install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -267,6 +267,7 @@ type ServerConfig struct {
 	DisableCompleteSnapshotJob        bool              `json:"disableCompleteSnapshotJob"`
 	InactivityPeriodForReposInDays    *int              `json:"inactivityPeriodForReposInDays"`
 	ShowSetupModal                    *bool             `json:"showSetupModal"`
+	isSingleOrgInstallation           *bool             `json:"isSingleOrgInstallation"`
```

Comment:
I'd recommend making this a non-pointer. If you do that, you get `false` as a default value and can save yourself an extra nil check when in practice nil doesn't really make sense here